packages/connect-query/src/unary-hooks.test.ts

```diff
@@ -170,7 +173,7 @@ describe('unaryHooks', () => {
       type ExpectType_CreateUserQueryOptionsParams0 = Expect<
         Equal<
           Parameters<typeof genCount.createUseQueryOptions>[0],
-          DisableQuery | PartialMessage<Message<CountRequest>> | undefined
+          DisableQuery | PartialMessage<CountRequest> | undefined
```

Comment:
One real change here. IT makes sense that we can just pass in CountRequest and don't need this double wrap.